src/app/components/receive/receive.component.html

```diff
@@ -200,8 +200,8 @@ <h3 class="uk-margin-remove-bottom">Incoming Transactions</h3>
             </ng-template>
             <td class="amount-column uk-text-middle uk-text-muted" [title]="('Incoming Transaction') + ( (pending.amountRaw && (pending.amountRaw > 0) ) ? ( ', +' + ( pending.amountRaw.toString(10) ) + ' raw' ) : '' )">
               <span class="uk-text-small">Ready to receive</span><br>
-              <span class="amount-integer">{{ pending.amount | rai: 'mnano,true' | amountsplit: 0 }}</span>
-              <span class="amount-fractional">{{ pending.amount | rai: 'mnano,true' | amountsplit: 1 }}</span>
+              <span class="amount-integer">{{ settings.settings.hideBalances ? '*' : pending.amount | rai: 'mnano,true' | amountsplit: 0 }}</span>
```

Comment:
not sure how many merchants will use the manual receives option, but they kinda do need to know the amount of nano they just received, without revealing balances of their other accounts (when using nault as a point-of-sale checkout)

Reply:
So perhaps we just leave the pending receives. I assume they would not have many pending?